pyproject.toml

```diff
@@ -43,7 +43,7 @@ jinja2 = "^3.1.2"
 huggingface-hub = "0.18.0"
 pyyaml = "^6.0"
 packaging = "^23.0"
-globus-compute-sdk = "^2.2.0"
```

Comment:
This was because a change to the globus compute client attribute names was breaking our tests, but outside of redoing how we mock the client I don't think there'd be any issues with 2.28.0